Client/Frontend/Fakespot/Views/AdjustRatingView.swift

```diff
@@ -0,0 +1,142 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+import Common
+import ComponentLibrary
+
+struct AdjustRatingViewModel {
+    let title: String
+    let description: String
+    let titleA11yId: String
+    let cardA11yId: String
+    let descriptionA11yId: String
+}
+
+class AdjustRatingView: UIView, Notifiable, ThemeApplicable {
+    private enum UX {
+        static let titleFontSize: CGFloat = 17
+        static let descriptionFontSize: CGFloat = 13
+        static let margins = UIEdgeInsets(top: 8, left: 8, bottom: 8, right: 8)
+        static let hStackSpacing: CGFloat = 4
+        static let vStackSpacing: CGFloat = 8
+    }
+
+    var rating: Double = 0.0 {
+        didSet {
+            starRatingView.rating = rating
+        }
+    }
+
+    private lazy var cardContainer: ShadowCardView = .build()
+    private lazy var contentView: UIView = .build()
+
+    var notificationCenter: NotificationProtocol = NotificationCenter.default
+
+    private lazy var titleLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.titleFontSize,
+                                                            weight: .medium)
+        label.numberOfLines = 0
+    }
+
+    private lazy var descriptionLabel: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
+                                                            size: UX.descriptionFontSize,
+                                                            weight: .regular)
+        label.numberOfLines = 0
+    }
+
+    private lazy var starRatingView: StarRatingView = .build()
+
+    private lazy var hStackView: UIStackView = .build { stackView in
+        stackView.axis = .horizontal
+        stackView.spacing = UX.hStackSpacing
+    }
+
+    private lazy var vStackView: UIStackView = .build { stackView in
+        stackView.axis = .vertical
+        stackView.spacing = UX.vStackSpacing
+        stackView.isLayoutMarginsRelativeArrangement = true
+        stackView.layoutMargins = UX.margins
+    }
+
+    private lazy var spacer: UIView = .build()
```

Comment:
We should not use spacers as they make it hard to see the current spacing and adjust spacing if the design changes.

Reply:
In this case I would need some help with this layout because I'm not sure how to create it differently. Could you help?

Reply:
![image](https://github.com/mozilla-mobile/firefox-ios/assets/26011662/904f6e6f-9038-46c5-ade7-b0be28469931)
To give some context, in vertical model the star view tries to fill al the available room and it looks like this. The spacer makes it take only the space it needs.